workspaces/blackduck/plugins/blackduck-backend/package.json

```diff
@@ -34,21 +34,21 @@
   },
   "dependencies": {
     "@backstage-community/plugin-blackduck-common": "workspace:^",
-    "@backstage/backend-defaults": "^0.7.0",
-    "@backstage/backend-plugin-api": "^1.1.1",
+    "@backstage/backend-defaults": "^0.8.0",
+    "@backstage/backend-plugin-api": "^1.2.0",
     "@backstage/config": "^1.3.2",
     "@backstage/errors": "^1.2.7",
     "@backstage/plugin-permission-common": "^0.8.4",
-    "@backstage/plugin-permission-node": "^0.8.7",
-    "express": "^4.17.1",
-    "express-promise-router": "^4.1.0"
+    "@backstage/plugin-permission-node": "^0.8.8",
+    "express": "^4.21.2",
+    "express-promise-router": "^4.1.1"
```

Comment:
These two changes aren't needed as they are handled by the `^`, can you please revert?